src/workerd/io/actor-cache.h

```diff
@@ -805,40 +805,40 @@ class ActorCacheOps::GetResultList {
 // forward-declared elsewhere.
 struct ActorCacheSharedLruOptions {
   // Memory usage that the LRU will try to stay under by evicting clean values.
-  size_t softLimit;
+  const size_t softLimit;
```

Comment:
I don't think these const declarations buy you anything.  Two reasons:
1. You can only set the values on construction, which seems like an unreasonable constraint on users of the struct.
2. It makes this struct non-moveable.

Reply:
I think I overshot a bit here :sweat_smile:, I wanted to see more `const` in the other parts of actor cache to make it easier to reason about, but that was tricky, so all that energy went into this struct https://github.com/cloudflare/workerd/pull/1914/commits/bdbae9bfac75544a05c4945c94ac33b5bd8640b1